testsite/requirements.txt

```diff
@@ -1,17 +1,32 @@
-Django==3.2.24
-djangorestframework==3.14.0
+# Unified requirements file supporting 3.7/3.2, 3.9/4.2, 3.12/5.1
+
+Django==3.2.24 ; python_version == "3.7"
```

Comment:
Is there a specific version of `pip` for this to work? If the feature has been there since 2020, then I guess I like this one better.